### ISSUE TITLE
package not publishing to NPM

### ISSUE THREAD

**Issue by CodebyOmar** [pascalgn/npm-publish-action#25]
The process isn't failing but i also can't see the package on NPM

**Comment by pascalgn** [comment]
Can you provide log output?

**Comment by pascalgn** [comment]
I think this is not yet fixed, or at least it's not clear what the cause for the problem is. However, it's a long time since there was some activity in this issue, so I will close it for now. If you have any new information or need any help, feel free to reopen this issue!

**Comment by calebfaruki** [comment]
I am having this issue when publishing to Github Package Repository. There is only one version previously published (0.0.1) on my repo. Before commiting to the main branch, I updated my package.json version to `0.0.2`. No tag or release was created.

```
name: npm-publish
on:
  push:
    branches:
      - main # Change this to your default branch
jobs:
  npm-publish:
    name: npm-publish
    runs-on: ubuntu-latest
    steps:
    - name: Checkout repository
      uses: actions/checkout@v2
    - name: Publish if version has been updated
      uses: pascalgn/npm-publish-action@1.3.5
      with: # All of theses inputs are optional
        tag_name: "%s"
        tag_message: "v%s"
        commit_pattern: "^Release (\\S+)"
        workspace: "."
        publish_command: "yarn"
        publish_args: "--non-interactive"
      env: # More info about the environment variables in the README
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # Leave this as is, it's automatically generated
        # NPM_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }} # You need to set this in your repo settings
```

Here is my GH action logs.

```
2020-12-05T09:51:23.9376525Z ##[section]Starting: Request a runner to run this job
2020-12-05T09:51:24.2005286Z Can't find any online and idle self-hosted runner in current repository that matches the required labels: 'ubuntu-latest'
2020-12-05T09:51:24.2005380Z Can't find any online and idle self-hosted runner in current repository's account/organization that matches the required labels: 'ubuntu-latest'
2020-12-05T09:51:24.2005739Z Found online and idle hosted runner in current repository's account/organization that matches the required labels: 'ubuntu-latest'
2020-12-05T09:51:24.3353578Z ##[section]Finishing: Request a runner to run this job
2020-12-05T09:51:32.4913680Z Current runner version: '2.274.2'
2020-12-05T09:51:32.4942617Z ##[group]Operating System
2020-12-05T09:51:32.4943457Z Ubuntu
2020-12-05T09:51:32.4943883Z 18.04.5
2020-12-05T09:51:32.4944405Z LTS
2020-12-05T09:51:32.4944844Z ##[endgroup]
2020-12-05T09:51:32.4945319Z ##[group]Virtual Environment
2020-12-05T09:51:32.4945864Z Environment: ubuntu-18.04
2020-12-05T09:51:32.4946570Z Version: 20201129.1
2020-12-05T09:51:32.4947408Z Included Software: https://github.com/actions/virtual-environments/blob/ubuntu18/20201129.1/images/linux/Ubuntu1804-README.md
2020-12-05T09:51:32.4948241Z ##[endgroup]
2020-12-05T09:51:32.4951181Z Prepare workflow directory
2020-12-05T09:51:32.5569135Z Prepare all required actions
2020-12-05T09:51:32.5578977Z Getting action download info
2020-12-05T09:51:32.9301589Z Download action repository 'actions/checkout@v2'
2020-12-05T09:51:34.5279255Z Download action repository 'pascalgn/npm-publish-action@1.3.5'
2020-12-05T09:51:35.4929087Z ##[group]Build container for action use: '/home/runner/work/_actions/pascalgn/npm-publish-action/1.3.5/Dockerfile'.
2020-12-05T09:51:35.4999830Z ##[command]/usr/bin/docker build -t 179394:4af0c2389b0b4b2895cb23f5951a27b0 -f "/home/runner/work/_actions/pascalgn/npm-publish-action/1.3.5/Dockerfile" "/home/runner/work/_actions/pascalgn/npm-publish-action/1.3.5"
2020-12-05T09:51:36.3073731Z Sending build context to Docker daemon  49.15kB
2020-12-05T09:51:36.3074209Z 
2020-12-05T09:51:37.2928859Z Step 1/9 : FROM node:12-alpine
2020-12-05T09:51:37.3031206Z  ---> 844f8bb6a3f8
2020-12-05T09:51:37.3059506Z Step 2/9 : LABEL "com.github.actions.name"="Publish to npm"
2020-12-05T09:51:37.3061022Z  ---> Running in f085122ec91d
2020-12-05T09:51:38.5711062Z Removing intermediate container f085122ec91d
2020-12-05T09:51:38.5711858Z  ---> 2fa5baf2007f
2020-12-05T09:51:38.5712586Z Step 3/9 : LABEL "com.github.actions.description"="Automatically publish new versions to npm"
2020-12-05T09:51:38.5992292Z  ---> Running in 38aaf89d0d1e
2020-12-05T09:51:39.0179660Z Removing intermediate container 38aaf89d0d1e
2020-12-05T09:51:39.0180749Z  ---> 6ec82a45cd8c
2020-12-05T09:51:39.0181726Z Step 4/9 : LABEL "com.github.actions.icon"="package"
2020-12-05T09:51:39.0400947Z  ---> Running in 868a4446ab5a
2020-12-05T09:51:40.0223214Z Removing intermediate container 868a4446ab5a
2020-12-05T09:51:40.0224342Z  ---> 099b830cbbf5
2020-12-05T09:51:40.0225152Z Step 5/9 : LABEL "com.github.actions.color"="blue"
2020-12-05T09:51:40.0423970Z  ---> Running in 2002ef9854fd
2020-12-05T09:51:41.0190440Z Removing intermediate container 2002ef9854fd
2020-12-05T09:51:41.0192055Z  ---> f4830216cb88
2020-12-05T09:51:41.0193478Z Step 6/9 : RUN apk add --no-cache git openssl
2020-12-05T09:51:41.0378031Z  ---> Running in 39ba7de664cc
2020-12-05T09:51:41.8596250Z fetch http://dl-cdn.alpinelinux.org/alpine/v3.11/main/x86_64/APKINDEX.tar.gz
2020-12-05T09:51:41.9756452Z fetch http://dl-cdn.alpinelinux.org/alpine/v3.11/community/x86_64/APKINDEX.tar.gz
2020-12-05T09:51:42.0776673Z (1/7) Installing ca-certificates (20191127-r2)
2020-12-05T09:51:42.0962814Z (2/7) Installing nghttp2-libs (1.40.0-r1)
2020-12-05T09:51:42.1040272Z (3/7) Installing libcurl (7.67.0-r1)
2020-12-05T09:51:42.1143724Z (4/7) Installing expat (2.2.9-r1)
2020-12-05T09:51:42.1217287Z (5/7) Installing pcre2 (10.34-r1)
2020-12-05T09:51:42.1349909Z (6/7) Installing git (2.24.3-r0)
2020-12-05T09:51:42.3020737Z (7/7) Installing openssl (1.1.1g-r0)
2020-12-05T09:51:42.3162152Z Executing busybox-1.31.1-r9.trigger
2020-12-05T09:51:42.3309942Z Executing ca-certificates-20191127-r2.trigger
2020-12-05T09:51:42.3832610Z OK: 24 MiB in 23 packages
2020-12-05T09:51:43.5045331Z Removing intermediate container 39ba7de664cc
2020-12-05T09:51:43.5045995Z  ---> e8b3bc6d6672
2020-12-05T09:51:43.5046728Z Step 7/9 : COPY . /tmp/src/
2020-12-05T09:51:44.0249989Z  ---> 358ca031af80
2020-12-05T09:51:44.0250822Z Step 8/9 : RUN yarn global add --production true "file:/tmp/src" && rm -rf /tmp/src
2020-12-05T09:51:44.0432266Z  ---> Running in d57b1a25bc78
2020-12-05T09:51:47.9145482Z yarn global v1.22.5
2020-12-05T09:51:47.9650293Z [1/4] Resolving packages...
2020-12-05T09:51:47.9781384Z [2/4] Fetching packages...
2020-12-05T09:51:47.9975604Z [3/4] Linking dependencies...
2020-12-05T09:51:48.1820139Z [4/4] Building fresh packages...
2020-12-05T09:51:48.1892794Z success Installed "npm-publish-action@0.0.1" with binaries:
2020-12-05T09:51:48.1894198Z       - npm-publish-action
2020-12-05T09:51:48.1938272Z Done in 0.28s.
2020-12-05T09:51:49.0221096Z Removing intermediate container d57b1a25bc78
2020-12-05T09:51:49.0221624Z  ---> 02050db6e9d6
2020-12-05T09:51:49.0222077Z Step 9/9 : ENTRYPOINT [ "npm-publish-action" ]
2020-12-05T09:51:49.0429473Z  ---> Running in 44bff827052a
2020-12-05T09:51:50.0201837Z Removing intermediate container 44bff827052a
2020-12-05T09:51:50.0202483Z  ---> ceb8ce1b40a4
2020-12-05T09:51:50.0204978Z Successfully built ceb8ce1b40a4
2020-12-05T09:51:50.0274272Z Successfully tagged 179394:4af0c2389b0b4b2895cb23f5951a27b0
2020-12-05T09:51:50.0295329Z ##[endgroup]
2020-12-05T09:51:50.0745019Z ##[group]Run actions/checkout@v2
2020-12-05T09:51:50.0745552Z with:
2020-12-05T09:51:50.0746214Z   repository: calebfaruki/myrepo
2020-12-05T09:51:50.0747248Z   token: ***
2020-12-05T09:51:50.0747569Z   ssh-strict: true
2020-12-05T09:51:50.0747961Z   persist-credentials: true
2020-12-05T09:51:50.0748501Z   clean: true
2020-12-05T09:51:50.0748799Z   fetch-depth: 1
2020-12-05T09:51:50.0749092Z   lfs: false
2020-12-05T09:51:50.0749400Z   submodules: false
2020-12-05T09:51:50.0749709Z ##[endgroup]
2020-12-05T09:51:50.5140952Z Syncing repository: calebfaruki/myrepo
2020-12-05T09:51:50.5142353Z ##[group]Getting Git version info
2020-12-05T09:51:50.5143582Z Working directory is '/home/runner/work/design/design'
2020-12-05T09:51:50.5144386Z [command]/usr/bin/git version
2020-12-05T09:51:50.5448872Z git version 2.29.2
2020-12-05T09:51:50.5499865Z ##[endgroup]
2020-12-05T09:51:50.5508991Z Deleting the contents of '/home/runner/work/design/design'
2020-12-05T09:51:50.5511362Z ##[group]Initializing the repository
2020-12-05T09:51:50.5515202Z [command]/usr/bin/git init /home/runner/work/design/design
2020-12-05T09:51:50.5623983Z Initialized empty Git repository in /home/runner/work/design/design/.git/
2020-12-05T09:51:50.5660898Z [command]/usr/bin/git remote add origin https://github.com/calebfaruki/myrepo
2020-12-05T09:51:50.5691759Z ##[endgroup]
2020-12-05T09:51:50.5692877Z ##[group]Disabling automatic garbage collection
2020-12-05T09:51:50.5697041Z [command]/usr/bin/git config --local gc.auto 0
2020-12-05T09:51:50.5738282Z ##[endgroup]
2020-12-05T09:51:50.5746374Z ##[group]Setting up auth
2020-12-05T09:51:50.5792934Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2020-12-05T09:51:50.5827255Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :
2020-12-05T09:51:50.6270241Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2020-12-05T09:51:50.6321267Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :
2020-12-05T09:51:50.6581787Z [command]/usr/bin/git config --local http.https://github.com/.extraheader AUTHORIZATION: basic ***
2020-12-05T09:51:50.6633543Z ##[endgroup]
2020-12-05T09:51:50.6635050Z ##[group]Fetching the repository
2020-12-05T09:51:50.6644895Z [command]/usr/bin/git -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=1 origin +2321f9dd49b110269f4c6d902d9d35f3d60a8a9d:refs/remotes/origin/main
2020-12-05T09:51:51.3919987Z remote: Enumerating objects: 39, done.        
2020-12-05T09:51:51.3929102Z remote: Counting objects:   2% (1/39)        
2020-12-05T09:51:51.3932606Z remote: Counting objects:   5% (2/39)        
2020-12-05T09:51:51.3933338Z remote: Counting objects:   7% (3/39)        
2020-12-05T09:51:51.3934261Z remote: Counting objects:  10% (4/39)        
2020-12-05T09:51:51.3935012Z remote: Counting objects:  12% (5/39)        
2020-12-05T09:51:51.3935577Z remote: Counting objects:  15% (6/39)        
2020-12-05T09:51:51.3936244Z remote: Counting objects:  17% (7/39)        
2020-12-05T09:51:51.3936758Z remote: Counting objects:  20% (8/39)        
2020-12-05T09:51:51.3937412Z remote: Counting objects:  23% (9/39)        
2020-12-05T09:51:51.3938051Z remote: Counting objects:  25% (10/39)        
2020-12-05T09:51:51.3938503Z remote: Counting objects:  28% (11/39)        
2020-12-05T09:51:51.3938966Z remote: Counting objects:  30% (12/39)        
2020-12-05T09:51:51.3939432Z remote: Counting objects:  33% (13/39)        
2020-12-05T09:51:51.3939883Z remote: Counting objects:  35% (14/39)        
2020-12-05T09:51:51.3940332Z remote: Counting objects:  38% (15/39)        
2020-12-05T09:51:51.3940766Z remote: Counting objects:  41% (16/39)        
2020-12-05T09:51:51.3941335Z remote: Counting objects:  43% (17/39)        
2020-12-05T09:51:51.3941803Z remote: Counting objects:  46% (18/39)        
2020-12-05T09:51:51.3942253Z remote: Counting objects:  48% (19/39)        
2020-12-05T09:51:51.3942690Z remote: Counting objects:  51% (20/39)        
2020-12-05T09:51:51.3943122Z remote: Counting objects:  53% (21/39)        
2020-12-05T09:51:51.3943756Z remote: Counting objects:  56% (22/39)        
2020-12-05T09:51:51.3944201Z remote: Counting objects:  58% (23/39)        
2020-12-05T09:51:51.3944884Z remote: Counting objects:  61% (24/39)        
2020-12-05T09:51:51.3945363Z remote: Counting objects:  64% (25/39)        
2020-12-05T09:51:51.3946031Z remote: Counting objects:  66% (26/39)        
2020-12-05T09:51:51.3946502Z remote: Counting objects:  69% (27/39)        
2020-12-05T09:51:51.3947186Z remote: Counting objects:  71% (28/39)        
2020-12-05T09:51:51.4393946Z remote: Counting objects:  74% (29/39)        
2020-12-05T09:51:51.4413409Z remote: Counting objects:  76% (30/39)        
2020-12-05T09:51:51.4413868Z remote: Counting objects:  79% (31/39)        
2020-12-05T09:51:51.4414472Z remote: Counting objects:  82% (32/39)        
2020-12-05T09:51:51.4416928Z remote: Counting objects:  84% (33/39)        
2020-12-05T09:51:51.4417817Z remote: Counting objects:  87% (34/39)        
2020-12-05T09:51:51.4418267Z remote: Counting objects:  89% (35/39)        
2020-12-05T09:51:51.4418903Z remote: Counting objects:  92% (36/39)        
2020-12-05T09:51:51.4419929Z remote: Counting objects:  94% (37/39)        
2020-12-05T09:51:51.4420747Z remote: Counting objects:  97% (38/39)        
2020-12-05T09:51:51.4421334Z remote: Counting objects: 100% (39/39)        
2020-12-05T09:51:51.4421791Z remote: Counting objects: 100% (39/39), done.        
2020-12-05T09:51:51.4422479Z remote: Compressing objects:   3% (1/32)        
2020-12-05T09:51:51.4423119Z remote: Compressing objects:   6% (2/32)        
2020-12-05T09:51:51.4423559Z remote: Compressing objects:   9% (3/32)        
2020-12-05T09:51:51.4424189Z remote: Compressing objects:  12% (4/32)        
2020-12-05T09:51:51.4424650Z remote: Compressing objects:  15% (5/32)        
2020-12-05T09:51:51.4425085Z remote: Compressing objects:  18% (6/32)        
2020-12-05T09:51:51.4425535Z remote: Compressing objects:  21% (7/32)        
2020-12-05T09:51:51.4425969Z remote: Compressing objects:  25% (8/32)        
2020-12-05T09:51:51.4426417Z remote: Compressing objects:  28% (9/32)        
2020-12-05T09:51:51.4426855Z remote: Compressing objects:  31% (10/32)        
2020-12-05T09:51:51.4428142Z remote: Compressing objects:  34% (11/32)        
2020-12-05T09:51:51.4428587Z remote: Compressing objects:  37% (12/32)        
2020-12-05T09:51:51.4429208Z remote: Compressing objects:  40% (13/32)        
2020-12-05T09:51:51.4429673Z remote: Compressing objects:  43% (14/32)        
2020-12-05T09:51:51.4430122Z remote: Compressing objects:  46% (15/32)        
2020-12-05T09:51:51.4430583Z remote: Compressing objects:  50% (16/32)        
2020-12-05T09:51:51.4431212Z remote: Compressing objects:  53% (17/32)        
2020-12-05T09:51:51.4431878Z remote: Compressing objects:  56% (18/32)        
2020-12-05T09:51:51.4432310Z remote: Compressing objects:  59% (19/32)        
2020-12-05T09:51:51.4432761Z remote: Compressing objects:  62% (20/32)        
2020-12-05T09:51:51.4433197Z remote: Compressing objects:  65% (21/32)        
2020-12-05T09:51:51.4433792Z remote: Compressing objects:  68% (22/32)        
2020-12-05T09:51:51.4434237Z remote: Compressing objects:  71% (23/32)        
2020-12-05T09:51:51.4434843Z remote: Compressing objects:  75% (24/32)        
2020-12-05T09:51:51.4435293Z remote: Compressing objects:  78% (25/32)        
2020-12-05T09:51:51.4435726Z remote: Compressing objects:  81% (26/32)        
2020-12-05T09:51:51.4436177Z remote: Compressing objects:  84% (27/32)        
2020-12-05T09:51:51.4436610Z remote: Compressing objects:  87% (28/32)        
2020-12-05T09:51:51.4437059Z remote: Compressing objects:  90% (29/32)        
2020-12-05T09:51:51.4437572Z remote: Compressing objects:  93% (30/32)        
2020-12-05T09:51:51.4438365Z remote: Compressing objects:  96% (31/32)        
2020-12-05T09:51:51.4438816Z remote: Compressing objects: 100% (32/32)        
2020-12-05T09:51:51.4439272Z remote: Compressing objects: 100% (32/32), done.        
2020-12-05T09:51:51.6454215Z remote: Total 39 (delta 0), reused 29 (delta 0), pack-reused 0        
2020-12-05T09:51:51.6995518Z From https://github.com/calebfaruki/myrepo
2020-12-05T09:51:51.6998957Z  * [new ref]         2321f9dd49b110269f4c6d902d9d35f3d60a8a9d -> origin/main
2020-12-05T09:51:51.7025252Z ##[endgroup]
2020-12-05T09:51:51.7026211Z ##[group]Determining the checkout info
2020-12-05T09:51:51.7027740Z ##[endgroup]
2020-12-05T09:51:51.7028441Z ##[group]Checking out the ref
2020-12-05T09:51:51.7034966Z [command]/usr/bin/git checkout --progress --force -B main refs/remotes/origin/main
2020-12-05T09:51:51.7238752Z Switched to a new branch 'main'
2020-12-05T09:51:51.7252599Z Branch 'main' set up to track remote branch 'main' from 'origin'.
2020-12-05T09:51:51.7253779Z ##[endgroup]
2020-12-05T09:51:51.7298002Z [command]/usr/bin/git log -1 --format='%H'
2020-12-05T09:51:51.7332810Z '2321f9dd49b110269f4c6d902d9d35f3d60a8a9d'
2020-12-05T09:51:51.7546224Z ##[group]Run pascalgn/npm-publish-action@1.3.5
2020-12-05T09:51:51.7547147Z with:
2020-12-05T09:51:51.7547446Z   tag_name: %s
2020-12-05T09:51:51.7547783Z   tag_message: v%s
2020-12-05T09:51:51.7548172Z   commit_pattern: ^Release (\S+)
2020-12-05T09:51:51.7548572Z   workspace: .
2020-12-05T09:51:51.7548931Z   publish_command: yarn
2020-12-05T09:51:51.7549392Z   publish_args: --non-interactive
2020-12-05T09:51:51.7549779Z env:
2020-12-05T09:51:51.7550950Z   GITHUB_TOKEN: ***
2020-12-05T09:51:51.7551299Z ##[endgroup]
2020-12-05T09:51:51.7587659Z ##[command]/usr/bin/docker run --name af0c2389b0b4b2895cb23f5951a27b0_58ef10 --label 179394 --workdir /github/workspace --rm -e GITHUB_TOKEN -e INPUT_TAG_NAME -e INPUT_TAG_MESSAGE -e INPUT_COMMIT_PATTERN -e INPUT_WORKSPACE -e INPUT_PUBLISH_COMMAND -e INPUT_PUBLISH_ARGS -e HOME -e GITHUB_JOB -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_REPOSITORY_OWNER -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_RETENTION_DAYS -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_SERVER_URL -e GITHUB_API_URL -e GITHUB_GRAPHQL_URL -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e GITHUB_ACTION_REPOSITORY -e GITHUB_ACTION_REF -e GITHUB_PATH -e GITHUB_ENV -e RUNNER_OS -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e ACTIONS_CACHE_URL -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/runner/work/design/design":"/github/workspace" 179394:4af0c2389b0b4b2895cb23f5951a27b0
2020-12-05T09:51:52.3181032Z Post job cleanup.
2020-12-05T09:51:52.4287096Z [command]/usr/bin/git version
2020-12-05T09:51:52.4347571Z git version 2.29.2
2020-12-05T09:51:52.4387760Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2020-12-05T09:51:52.4423693Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :
2020-12-05T09:51:52.4701375Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2020-12-05T09:51:52.4731640Z http.https://github.com/.extraheader
2020-12-05T09:51:52.4740681Z [command]/usr/bin/git config --local --unset-all http.https://github.com/.extraheader
2020-12-05T09:51:52.4779653Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :
2020-12-05T09:51:52.5097969Z Cleaning up orphan processes
```
